init-tools.sh

```diff
@@ -68,6 +68,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

Comment:
Is this going to cause problems for any of the arm efforts?  If not, looks fine to me.